### PR TITLE
Error: unknown command "ko" for "kubectl"

### PR DESCRIPTION
**Pull request by yanwei2019**
For issue #652

### CODE REVIEW

File: dist/images/install.sh [hunk 1]
```diff
@@ -2186,6 +2186,14 @@ chmod +x /usr/local/bin/kubectl-ko
 echo "-------------------------------"
 echo ""
 
+echo $PATH | grep '/usr/local/bin'
```
Comment (halfcrazy) [inline dist/images/install.sh:2189]: echo ":$PATH:" | grep -q ":/usr/local/bin:"

Comment (yanwei2019) [reply dist/images/install.sh:2189]: ok,thanks.

**Review by halfcrazy — APPROVED** [review]

**Comment by oilbeater** [comment]
@yanwei2019 can you also update the scripts to `dist/image/kubectl-ko` and  `dist/image/install-pre-1.16.sh` for consistency

**Comment by yanwei2019** [comment]
dist/image/install-pre-1.16.sh 可以一起修改
dist/image/kubectl-ko 应该没必要修改吧，PATH中增加/usr/local/bin就是为了找到kubectl-ko

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
Thanks! @yanwei2019

**Pull request merged**